doc/rtd/howto/debug_user_data.rst

```diff
@@ -0,0 +1,37 @@
+How to debug user data
+======================
+
+Two of the most common issues with cloud config user data are:
+
+1. Incorrectly formatted YAML
+2. The first line does not contain ``#cloud-config``
+
+Static user data validation
+---------------------------
+
+To verify your cloud config is valid YAML you can use `validate-yaml.py`_.
+
+To ensure the keys and values in your user data are correct, you can run:
+
+.. code-block:: shell-session
+
+    cloud-init schema --system --annotate
```

Comment:
This `--system` requires root user to normal user can't do this. let's provide sudo here
```suggestion
    sudo cloud-init schema --system --annotate
```